lib/couchbase/options.rb

```diff
@@ -1146,6 +1146,22 @@ def initialize(get_options: Get.new,
     end
 
     # Options for {Couchbase::Cluster.connect}
+    #
+    # @example Pass authenticator object to Options
+    #   Cluster.connect("couchbase://localhost",
+    #     Options::Cluster(authenticator: PasswordAuthenticator.new("Administrator", "password")))
+    #
+    # @example Shorter version, more useful for interactive sessions
+    #   Cluster.connect("couchbase://localhost", "Administrator", "password")
+    #
+    # @example Authentication with TLS client certificate (note +couchbases://+ schema)
+    #   Cluster.connect("couchbases://localhost?trust_certificate=/tmp/ca.pem",
+    #     Options::Cluster(authenticator: CertificateAuthenticator.new("/tmp/certificate.pem", "/tmp/private.key")))
```

Comment:
Should there be an `@example` for timeouts here too?
(Or not, as that's a less important use-case than these?
Not important question, just wanted to double-check, as seemed incongruous to be adding a feature and docs about *something other than that same feature*)